fluent-plugin-logdna.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.require_paths = ["lib"]
   s.files         = `git ls-files -z`.split("\x0")
   s.executables   = s.files.grep(%r{^bin/}) { |f| File.basename(f) }
-  s.required_ruby_version = Gem::Requirement.new(">= 2.5.0")
-  s.add_runtime_dependency "fluentd", ">= 0.12.0", "< 2"
+  s.required_ruby_version = Gem::Requirement.new(">= 2.0.0")
```

Comment:
why are we lowering this?

Reply:
also does this have an associated Jira task?

Reply:
@dm36, well not anymore